course/frontend/array-shuffle/index.html

```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```

Comment:
WARN! you have conflictive code here! remove the lines
'=====
and
'>>>> upstream/develop